backend/api/Models/BatchUpdate/Reply.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+
+namespace Pims.Api.Models.BatchUpdate
+{
+    /// <summary>
+    /// Provides a wrapper for batch update replies.
+    /// </summary>
+    public class Reply<T>
+    {
+        public Reply(IList<T> payload)
+        {
+            Payload = payload;
+        }
+
+        /// <summary>
+        /// get/set - The updated payload.
+        /// </summary>
+        public IList<T> Payload { get; set; }
+
+        /// <summary>
+        /// get/set - Batch error messages if any.
+        /// </summary>
+        public IList<string> ErrorMessages { get; set; } = new List<string>();
```

Comment:
These aren't used currently correct? what is the use case for this?

Reply:
We should chat about this too. Id does not look like we have a way of passing errors from each layer of the application. Ideally, we would have db errors bubbling up to services that could tell us further information about the processing. This is the first step to making those.

Reply:
Does ErrorHandlingMiddleware.cs partially meet this need or no?

Reply:
For logging, yes, but it does not look like is raising those errors cleanly to the front end. We could chat more about this later